bundle/Resources/public/js/views/plugins/cof-copycontentplugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-copycontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the Copy Content plugin.
+     *
+     * @module cof-copycontentplugin
+     */
+    Y.namespace('cof.Plugin');
+
+    /**
+     * Provides plugin for the copy content plugin.
+     *
+     * @namespace cof.Plugin
+     * @class CopyContent
+     * @constructor
+     * @extends eZ.Plugin.CopyContent
+     */
+    Y.cof.Plugin.CopyContent = Y.Base.create('cofCopyContentPlugin', Y.eZ.Plugin.CopyContent, [], {
```

Comment:
using an "error" notification for that message is maybe not the best option. I would integrate the error message near the Content Type selector also the message is wrong (it's not necessarily a 'isContainer' issue), I would write something like: *The <Content Type name> Content item was correctly created but unfortunately it can not be used here, please choose or create another Content item*.
Again, I agree it's not super nice UI wise, but it's the best we can do at the moment to fix the bug in a reliable way.